lib/extractor/index.ts

```diff
@@ -196,7 +196,12 @@ function layersWithLatestFileModifications(
 
       // not adding the original file to extractedLayers
       // and removing it from the set since it can be found in consecutive layers
-      if (removedFilesToIgnore.has(filename)) {
+      if (
+        removedFilesToIgnore.has(filename) ||
+        Array.from(removedFilesToIgnore).some((removedFile) =>
+          filename.startsWith(removedFile),
+        )
```

Comment:
What happens if there are multiple files in the deleted directory? 

It looks like this would work only for the first file since we have `removedFilesToIgnore.delete(filename);` when handling it.